bin/picca_deltas.py

```diff
@@ -494,50 +494,26 @@ def main():
                 forest.order = args.order
 
     ### Read masks
-    mask_obs_frame = None
-    mask_rest_frame = None
-    mask_rest_frame_dla = None
     if args.mask_file is not None:
         args.mask_file = os.path.expandvars(args.mask_file)
         try:
             mask = Table.read(args.mask_file,
                               names=('type', 'wave_min', 'wave_max', 'frame'),
                               format='ascii')
-            if 'OBS' in mask['frame']:
-                w = mask['frame'] == 'OBS'
-                mask_obs_frame = np.array([
-                    np.log10(mask['wave_min'])[w],
-                    np.log10(mask['wave_max'])[w]
-                ]).T
-            else:
-                mask_obs_frame = np.array([])
-            if 'RF' in mask['frame']:
-                w = mask['frame'] == 'RF'
-                mask_rest_frame = np.array([
-                    np.log10(mask['wave_min'])[w],
-                    np.log10(mask['wave_max'])[w]
-                ]).T
-            else:
-                mask_rest_frame = np.array([])
-            if 'RF_DLA' in mask['frame']:
-                w = mask['frame'] == 'RF_DLA'
-                mask_rest_frame_dla = np.array([
-                    np.log10(mask['wave_min'])[w],
-                    np.log10(mask['wave_max'])[w]
-                ]).T
-            else:
-                mask_rest_frame_dla = None
+            col_log_min=Column(np.log10(mask['wave_min']),name='log_wave_min')
+            col_log_max=Column(np.log10(mask['wave_max']),name='log_wave_max')
+            mask.add_columns([col_log_min,col_log_max])
```

Comment:
Thanks for these changes. This can be even simpler thanks to the dictionary-like behaviour of Tables. You can do:
```
mask['log_wave_min'] = np.log10(mask['wave_min'])
mask['log_wave_max'] = np.log10(mask['wave_max'])
```

Reply:
indeed, this is even better, done!